src/app/components/UserTos.js

```diff
@@ -61,59 +59,35 @@ const UserTosComponent = (props) => {
   const actions = [
     // eslint-disable-next-line jsx-a11y/click-events-have-key-events
     <div onClick={handleValidate.bind(this)} style={{ cursor: 'pointer' }}>
-      <Button
-        id="tos__save"
-        color="primary"
+      <ButtonMain
+        buttonProps={{
+          id: 'tos__save',
+        }}
+        size="default"
+        variant="contained"
+        theme="brand"
         onClick={handleSubmit.bind(this)}
         disabled={!checkedTos}
-      >
-        <FormattedMessage id="userTos.save" defaultMessage="Save" description="Button label for the user to save their review of the terms of service" />
-      </Button>
+        label={
+          <FormattedMessage id="userTos.save" defaultMessage="Save" description="Button label for the user to save their review of the terms of service" />
+        }
+      />
     </div>,
   ];
 
-  const linkStyle = {
-    textDecoration: 'underline',
-  };
-
-  const communityGuidelinesLink = (
```

Comment:
reviewers note: this link 404s, so it was removed, I don't believe it is necessary anymore